tests/testthat/test-qenv_eval_code.R

```diff
@@ -15,14 +15,14 @@ testthat::test_that("eval_code doesn't have access to environment where it's cal
   )
 })
 
-testthat::test_that("@env in qenv is always a sibling of .GlobalEnv", {
+testthat::test_that("@.xData in qenv is always a sibling of .GlobalEnv", {
   q1 <- qenv()
-  testthat::expect_identical(parent.env(q1@env), parent.env(.GlobalEnv))
+  testthat::expect_identical(parent.env(q1@.xData), parent.env(.GlobalEnv))
 
   q2 <- eval_code(q1, quote(a <- 1L))
-  testthat::expect_identical(parent.env(q2@env), parent.env(.GlobalEnv))
+  testthat::expect_identical(parent.env(q2@.xData), parent.env(.GlobalEnv))
   q3 <- eval_code(q2, quote(b <- 2L))
-  testthat::expect_identical(parent.env(q3@env), parent.env(.GlobalEnv))
+  testthat::expect_identical(parent.env(q3@.xData), parent.env(.GlobalEnv))
 })
```

Comment:
duplicated test - already introduced in line 52 
```suggestion
```

Reply:
1e256810